.gitignore

```diff
@@ -1 +1,4 @@
 *.pyc
+config.py
```

Comment:
Would you mind updating the readme to say to create a `config.py` and whizz in an example of what it should contain? Also, I've just thought - whack yourself in the [credits](https://github.com/bobrippling/lunchbot/blob/master/PS4Bot.py#L391). The emoji choice is all yours